db/comdb2.c

```diff
@@ -2123,14 +2127,18 @@ static int llmeta_load_tables(struct dbenv *dbenv, char *dbname, void *tran)
         }
         free(csc2text);
         csc2text = NULL;
-        tbl = newdb_from_schema(dbenv, tblnames[i], NULL, dbnums[i], i, 0);
+        nstripes = db_get_dtastripe_by_name(tblnames[i], tran);
+        is_systable = db_get_is_systable_by_name(tblnames[i], tran);
+        tbl = newdb_from_schema(dbenv, tblnames[i], NULL, dbnums[i], i, 0, nstripes);
         if (tbl == NULL) {
             logmsg(LOGMSG_ERROR, "newdb_from_schema failed %s:%d\n", __FILE__,
                     __LINE__);
             rc = 1;
             break;
         }
         tbl->schema_version = ver;
```

Comment:
What if we passed is_systable as a parameter to newdb_from_schema()? 
You don't need tbl->nstripes = nstripes; because nstripes is already set.